src/silica.js

```diff
@@ -641,6 +641,9 @@ window["Silica"] = {
     if (!this.dataset["noStopPropagation"]) {
       evt.stopPropagation();
     }
+    // Readonly current target is only available when the event is handled thus
+    // pass along in another obj for client usage
+    evnt._current_target = evnt.currentTarget
     PubSub.Pub("SiO2-HREF", evt);
```

Comment:
Variable name is incorrect, you have `evnt` instead of `evt`

Instead of storing it as a "private" field, just pass it as a second argument to the `Pub`
`PubSub.Pub("SiO2-HREF", evt, evt.currentTarget)`

If you can make this tweak, I will merge and release a new RC